src/wallet/wallet.cpp

```diff
@@ -3530,6 +3530,22 @@ bool CWallet::GetNewDestination(const OutputType type, const std::string label,
     return true;
 }
 
+bool CWallet::GetNewChangeDestination(const OutputType type, CTxDestination& dest, std::string& error)
+{
```

Comment:
nit: same comment about clearing the error string

Reply:
Done

Reply:
Done